src/docs/product/relay/options.mdx

```diff
@@ -74,6 +74,18 @@ The following document the general settings for Relay:
 
   Password for the PKCS12 archive in `relay.tls_identity_path`.
 
+`relay.override_project_ids`
+
+: _Boolean, optional_
+
+  Override project identifiers stated in client DSNs with the identifier used at
```

Comment:
Making this edit based on the way the other options are written where the option is the implied (singular) subject of the sentence. 

```suggestion
  Overrides project identifiers stated in client DSNs with the identifier used at
```